lib/rudder/analytics/defaults.rb

```diff
@@ -7,11 +7,9 @@ module Request
         HOST = 'localhost'
         PORT = 8080
         PATH = '/v1/batch'
-        DATA_PLANE_URL = 'http://localhost:8080/v1/batch'
-        SSL = false
         HEADERS = { 'Accept' => 'application/json',
                     'Content-Type' => 'application/json',
-                    'User-Agent' => "rudderanalytics-ruby/#{Analytics::VERSION}" }
```

Comment:
is this info now part of the event.context.library?

Reply:
We don't send User-Agent manually in any other SDK. This line is removed.